pom.xml

```diff
@@ -77,7 +86,8 @@ under the License.
         <zookeeper.version>3.4.6</zookeeper.version>
 
         <pig.version>0.9.2</pig.version> <!-- Newest: 0.15.0 -->
-
+        
+        <!-- Can we move geomesa to indexing? -->
```

Comment:
Here is the reason it is still WIP.  To do, remove dependencies.  I already copied them to the indexing pom.xml.